presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -478,6 +479,10 @@ std::string SystemConfig::memoryArbitratorKind() const {
   return optionalProperty<std::string>(kMemoryArbitratorKind).value_or("");
 }
 
+bool SystemConfig::memoryArbitratorGlobalArbitrationEnabled() const {
+  return optionalProperty<bool>(kMemoryArbitratorGlobalArbitrationEnabled).value_or("");
```

Comment:
.value_or(false)

Reply:
All the system configs in Prestissimo doesn't provide the default here. I follow the same pattern.

Reply:
Yeah but you are assigning a string to a bool